test.hxml

```diff
@@ -0,0 +1,14 @@
+## Neko
+-main TestMain
+-lib munit
+-lib hamcrest
+-lib svg
+-lib openfl
+-lib lime
+
+-cp src
+-cp test
+-neko build/neko_test.n
+-D lime
+-D lime_cairo
+--remap flash:openfl
```

Comment:
I'd add `-cmd neko build/neko_test.n`,

note that adding neko to the filename isn't necessary since the .n extension is explicit.